internal/api/v1/servicebindings.go

```diff
@@ -155,6 +186,22 @@ func (hc ServicebindingsController) Delete(w http.ResponseWriter, r *http.Reques
 		return OrgIsNotKnown(org)
 	}
 
+	apiErr := DeleteBinding(ctx, cluster, org, appName, serviceName, username)
+	if apiErr != nil {
+		return apiErr
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	_, err = w.Write([]byte{})
```

Comment:
Consider using `jsonResponse` from this PR (now merged): https://github.com/epinio/epinio/pull/811/files#diff-168b4a07f391b22def39fd870b0373811deb114cd513793f9cf5a3182f8452ceR81

Reply:
Yes, that is part of what I have to do as part of the rebase / conflict resolution.